usr/src/test/smartos-test/README

```diff
@@ -54,7 +60,11 @@ Specifically, the script will:
   file that need to be installed to a normally read-only location, and
   extract the portions of test archive that must appear in /usr
 * extract the remaining test archive contents to /opt and /kernel
-* configure pkgsrc
+* configure pkgsrc (or use existing GZ pkgsrc if installed)
```

Comment:
I think here I would say:

* install pkgsrc-tools (or use existing GZ pkgsrc if installed)

Reply:
s/use/update/g but otherwise yes, I like this rewording.